storage/client/client.go

```diff
@@ -215,6 +215,9 @@ func (c *StorageClient) Status(ctx context.Context) (*Status, error) {
 	).Scan(&s.LatestNodeBlock)
 	switch err {
 	case nil:
+		if s.LatestNodeBlock < s.LatestBlock {
```

Comment:
Nit: Maybe a comment since it's not obvious why this is needed?
```
// Current node height is fetched in a separate goroutine, so it's possible for it
// to be more out of date than the height of the most recently processed block.
```